doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -237,44 +237,54 @@ analysis on inductive or co-inductive objects (see :ref:`variants`).
 Induction
 ---------
 
-.. tacn:: induction @term
-   :name: induction
+.. tacn:: induction {+, @induction_clause } {? @induction_principle }
 
-   This tactic applies to any goal. The argument :n:`@term` must be of
-   inductive type and the tactic :n:`induction` generates subgoals, one for
-   each possible form of :n:`@term`, i.e. one for each constructor of the
-   inductive type.
+   Applies an :term:`induction principle` to generate a subgoal for
+   each constructor of an inductive type.
 
-   If the argument is dependent in either the conclusion or some
+   If the argument is :term:`dependent <dependent product>` in the conclusion or some
    hypotheses of the goal, the argument is replaced by the appropriate
-   constructor form in each of the resulting subgoals and induction
+   constructor in each of the resulting subgoals and induction
    hypotheses are added to the local context using names whose prefix
-   is **IH**.
-
-   There are particular cases:
-
-   + If term is an identifier :n:`@ident` denoting a quantified variable of the
-     conclusion of the goal, then inductionident behaves as :n:`intros until
-     @ident; induction @ident`. If :n:`@ident` is not anymore dependent in the
-     goal after application of :n:`induction`, it is erased (to avoid erasure,
-     use parentheses, as in :n:`induction (@ident)`).
-   + If :n:`@term` is a :n:`@natural`, then :n:`induction @natural` behaves as
+   is **IH**.  The tactic is similar to :tacn:`destruct`, except that
+   `destruct` doesn't generate induction hypotheses.
+
+   There are several cases:
```

Comment:
Note that none of these is specific to `induction` (this is a repetition of what is described in `destruct` so I don't know if we want to keep it). BTW, I like this description with bullet points more than the equivalent documentation scattered around for `destruct` so I would even suggest using this form above.

Reply:
When the behavior is identical, it's probably better not to repeat it.  So I dropped the text of your previous comment.  If the others were dropped, you would need to mentally substitute "induction" for "destruct" in most of them, so I thought it better to repeat them.  I could add a sentence here before the points saying they're repetitions but with "induction" instead of "destruct".  destruct doesn't describe the induction principle, so that should stay.

I agree that bullet points look nicer, but the other format makes it much easier to scan for elements of the syntax.  I suppose you could have bullet points with the form `. <syntax>: text`, but not sure that would be an improvement.  Would you want to do this change pervasively (and when) or just for some tactics/commands?

Your thoughts?

Reply:
OTOH, if "induction" is a very commonly used tactic, that would be a small point in favor of repetition.

Reply:
Yes, `induction` is a very commonly used tactic, but so is `destruct` and I prefer having a clear link to `destruct` for the description of how it treats its argument rather than a repetition.

No, I don't want to pervasively go for bullet points in the manual. I quite like the way that we have decided to document syntax elements. However, this case is a bit different IMHO because the bullet points represent *alternatives* and some alternatives are not even part of the cases that one can see from the syntax (`ident`, which is a sub-case of `one_term`).

So what I suggested was to reuse the way it was in the description of `induction`, i.e. first the alternative ways of providing the main argument (with bullet points) then the other syntax elements (with the usual style for describing those).

Reply:
OK, we're agreed on the style, but I'm still not sure what change you want me to make. Do you want me to copy the bullet points to `destruct` with substitution of `destruct` as needed?  Or what?

Reply:
> Do you want me to copy the bullet points to `destruct` with substitution of `destruct` as needed?

Yes, something like that (+ the clean-up of the redundant info that this would create).